task2/client/client.go

```
@@ -0,0 +1,48 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"net"
+	"os"
+	"strings"
+)
+
+func main() {
+	conn, err := net.Dial("tcp", ":8083")
+	if err != nil {
+		panic(err)
+	}
+
+	fmt.Println("connected to ", conn.RemoteAddr().String())
+	defer fmt.Println("disconnected from ", conn.RemoteAddr().String())
+	defer conn.Close()
+
+	for {
+		input := bufio.NewReader(os.Stdin)
+
+		fmt.Println("enter data to send")
+
+		data, err := input.ReadString('\n')
+
+		if err != nil {
```

Please remove all extra empty lines.

---

Please do not ignore an error.